File: CS#/Rectangle.cs
using System;

class Rectangle
{
    private double width;
    private double height;

    public double Width
    {
        get { return width; }
        private set { width = value; }
    }

    public double Height
    {
        get { return height; }
        private set { height = value; }
    }

    public Rectangle(double width, double height)
    {
        Width = width;
        Height = height;
    }

    public double area()
    {
        return width * height;
    }

    public void printInfo()
    {
        Console.WriteLine("Rectangle");
        Console.WriteLine("Width = " + width);
        Console.WriteLine("Height = " + height);
        Console.WriteLine("Area = " + area());
    }
}

File: Python/Rectangle.py
class Rectangle:
    # - width : Real
    # - height : Real
    def __init__(self, width, height):
        self.__width = width
        self.__height = height

    @property
    def width(self):
        return self.__width

    @property
    def height(self):
        return self.__height

    # + area() : Real
    def area(self):
        return self.__width * self.__height

    # + printInfo()
    def printInfo(self):
        print("Rectangle")
        print("Width =", self.__width)
        print("Height =", self.__height)
        print("Area =", self.area())
